pandas/core/nanops.py

```diff
@@ -344,12 +344,22 @@ def nanmedian(values, axis=None, skipna=True):
 
     values, mask, dtype, dtype_max = _get_values(values, skipna)
 
+    if not skipna:
```

Comment:
Isn't the desired behavior for nanmedian to be like median when skipna is turned off though? If not, then what is the skipna flag supposed to do?